extensions/amp-story/1.0/amp-story-open-page-attachment.css

```diff
@@ -166,3 +166,56 @@
   text-shadow: 0px 0px 6px rgba(0, 0, 0, 0.36) !important;
   white-space: nowrap !important;
 }
+
+/** For amp-story-inline-page-attachment-v2 experiment elements. */
+.i-amphtml-story-inline-page-attachment-chip {
+  --cta-chip-bg-color: rgba(255, 255, 255, 0.5);
+  background-color: var(--cta-chip-bg-color) !important;
+  display: flex !important;
+  border-radius: 24px !important;
+  align-items: center !important;
+  overflow: hidden !important;
+  margin-bottom: 12px !important;
+}
+
+.i-amphtml-story-inline-page-attachment-chip:only-child {
+  margin-bottom: 20px !important;
+}
+
+.i-amphtml-story-inline-page-attachment-img,
+.i-amphtml-story-inline-page-attachment-arrow {
+  vertical-align: middle !important;
+  background-size: contain !important;
+  background-repeat: no-repeat !important;
+}
+
+.i-amphtml-story-inline-page-attachment-img {
+  height: 48px !important;
+  width: 48px !important;
+  background-color: white !important;
```

Comment:
comment:
This could be a color variable. 
Let's do that in the theming PR.

Reply:
noted for theming PR - #32771